### PR TITLE
Noteupdate

### PR DESCRIPTION
**Pull request by stevendes**
Trying to update from the branch

### CODE REVIEW

**Review by ernestmc — APPROVED** [review]
lgtm

**Comment by eborghi10** [comment]
@stevendes when you create a PR add us as reviewers.

File: milner_desvars/diff-drive.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# how a differential-drive robot works and how to control it
```
Comment (eborghi10) [inline milner_desvars/diff-drive.md]: Start titles with uppercase letters.

File: milner_desvars/diff-drive.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+v_l = counter-clockwise angular velocity of left wheel (ie radians per second)
+~~~~
+
+Because the Unicycle model is easy to design around it, and fortunately for us, we can convert unicycle inputs into differential inputs, for that we have to use some values of the geometry of the wheeled robot, this values are:
+
+~~~~
+L = wheelbase (in meters per radian)
+
+R = wheel radius (in meters per radian
```
Comment (eborghi10) [inline milner_desvars/diff-drive.md]: Missing ending parenthesis.

File: milner_desvars/diff-drive.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The numerator for both equations are in meters per second. The denominator is in meters per radian. Both v_r and v_l result in radians per second, clock-wise and counter-clock-wise respectively— what we would expect.
+
+
+# Unicycle model and Twist message
+
+The most common way to send movement commands to the robot is with use of geometry_msgs/Twist message type. Then motor driver node should use data stored in them to control the motor.
+
+The geometry_msgs/Twist message express velocity in free space and consists of two fields:
+
+* Vector3 linear - represents linear part of velocity [m/s]
```
Comment (eborghi10) [inline milner_desvars/diff-drive.md]: When you mention code use `this format`. You can do it in this way:

```
`This code will be formatted` and this don't
```

The words I'm referring to are:

- `Vector3`
- `linear.x`
- `Twist`
- ...

File: milner_desvars/gazebo.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Notes
+
+## Client and server separation
+
+When you run Gazebo you are running to different executables. In one hand 'gzserver' and in the other 'gzclient'.
+
+The 'gzserver' executable runs the physics update-loop and sensor data generation. This is the core of Gazebo and it can be used independently of a graphical interface.
```
Comment (eborghi10) [inline milner_desvars/gazebo.md]: Replace single quotes with inverted quotes to use [code formatting](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#quoting-code).

Comment (eborghi10) [reply milner_desvars/gazebo.md]: Apply this all over the text.

File: milner_desvars/gazebo.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+A model file uses the same SDF format as world files, but should only contain a single <model> ... <model>('<model>' is one of the elements you can have in an SDF file, others for example are 'model' and 'actor').The purpose of these files is to facilitate model reuse and simplify world files.
+
+'Ex. can.sdf' 
+
+## Environment Variables
+
+Gazebo uses several environment variables to locate files and set up communications between the server and clients. Default values that work for most cases are compiled in. Some environment variables are:
+
+*GAZEBO_MODE_PATH
```
Comment (eborghi10) [inline milner_desvars/gazebo.md]: This list is not correctly done. You need a space after the bullet.

**Review by eborghi10 — CHANGES_REQUESTED** [review]
I left some comments, PTAL.

File: milner_desvars/ros-notes.md [hunk 6]
```diff
@@ -0,0 +1,490 @@
+# ROS
```
Comment (ernestmc) [inline milner_desvars/ros-notes.md]: This file is for *this* week's assignment. Let's add the new assignment to a different pull-request. Also the rqt image shouldn't go in here.

**Review by eborghi10 — APPROVED** [review]
LGTM!

**Pull request merged**